src/modules/tfs.module.ts

```diff
@@ -15,6 +15,8 @@ import { FacilityLoanModule } from '@ukef/modules/facility-loan/facility-loan.mo
 import { PartyModule } from '@ukef/modules/party/party.module';
 import { PartyExternalRatingModule } from '@ukef/modules/party-external-rating/party-external-rating.module';
 
+import { FacilityLoanTransactionModule } from './facility-loan-transaction/facility-loan-transaction.module';
```

Comment:
Can we have the import be from `@ukef/modules` for consistency please?

Reply:
Changed.